raft_test.go

```diff
@@ -5030,3 +5030,10 @@ func newTestRawNode(id uint64, election, heartbeat int, storage Storage) *RawNod
 	}
 	return rn
 }
+
+// TestPayloadSizeOfEmptyEntry ensures that payloadSize of empty entry is always zero
+func TestPayloadSizeOfEmptyEntry(t *testing.T) {
+	if payloadSize(pb.Entry{Data: nil}) != 0 {
+		t.Fatalf("EmptyEntry expected size: %d, got: %d", 0, payloadSize(pb.Entry{Data: nil}))
```

Comment:
We've been moving the library over to use the `testify/require` library. So let's adjust this slightly:
```go
e := pb.Entry{Data: nil}
require.Equal(t, 0, payloadSize(e))
```